lib/plugins/TaskRunner/Drupal.js

```diff
@@ -0,0 +1,171 @@
+"use strict"
+
+module.exports = class Drupal extends require('./Script') {
+
+
+  /**
+   * Options (used by this task):
+   *   - clearCaches: (boolean) - Whether to clear all caches after the build is finished. Defaults to true.
```

Comment:
Let's stick to JSDoc conventions for specifying params.

For example:
```
 @param {Object} options - Options used by this task.
 @param {boolean} options.clearCaches - ...
```

More verbose, I know, but then we can generate docs.